src/sentry/conf/server.py

```diff
@@ -1336,6 +1336,11 @@ def create_partitioned_queues(name):
 # Rate limiting backend
 SENTRY_RATELIMITER = "sentry.ratelimits.base.RateLimiter"
 SENTRY_RATELIMITER_OPTIONS = {}
+SENTRY_RATELIMITER_DEFAULTS = {
+    "ip": (1000, 1),
```

Comment:
I'm anticipating bugs down the line when developers don't get these keys exactly right. Can we get an enum of acceptable categories? Doing this would make it more complicated for users to override these defaults in configs so you might also have to do something like:
```py
SENTRY_RATELIMITER_DEFAULTS = {
	RateLimitCategory.IP: (SENTRY_RATELIMITER_DEFAULT_IP, 1),
	RateLimitCategory.USER: (SENTRY_RATELIMITER_DEFAULT_USER, 1),
	RateLimitCategory.ORGANIZATION: (SENTRY_RATELIMITER_DEFAULT_ORG, 1),
	...
}
```